doc/tutorials/python-kdb.md

```diff
@@ -0,0 +1,39 @@
+# How-To: Python kdb
+
+## Introduction
+
+When programming in python it is possible to access the kdb database, changing values of existing keys, adding and deleting keys and a few other things.
+
+## First Steps
+
+In order to being able to use kdb, obviously, you at first need to `import kdb`. You need access to an python-object of kdb. This is accomlished by calling `kdb.KDB()` and saving this to a variable because later on this object will be needed for various operations.
+
+## Keyset
+
+A keyset is basically a list of the keys that lie within the specified range. At first it is necessairy to create a new keyset. When simply calling `kdb.KeySet()` the keyset is of undefined size. It is possible to specify a certain (maximum) size for a keyset. To load keys into to keyset from the database you simply call the method `get` provided by the kdb-object.
```

Comment:
This line is really long :-) Can you run `fmt`?

typo: necessary.

The size of a keyset created with `kdb.KeySet()` is 0 (its empty). I think you confused it with the allocation hint, which is not useful to discuss for such a tutorial.

Can you make the tutorial more walk-through, always give short python snippets that will work. You can start with an hello world ;)